grpc-api/src/main/java/io/deephaven/appmode/ApplicationFactory.java

```diff
@@ -0,0 +1,93 @@
+package io.deephaven.appmode;
+
+import com.google.rpc.Code;
+import io.deephaven.db.appmode.DynamicApplication;
```

Comment:
packages are weird - why does appmode, in the appmode project, use the io.deephaven.db.appmode package, while not depending on db, but we have io.deephaven.appmode in the grpc-api project?

can we move this to grpc-api, and remove db from packages in appmode (that are explicitly not part of db)?